src/modules/0.0.22/aws_acm_list/index.ts

```diff
@@ -19,19 +20,22 @@ class CertificateMapper extends MapperBase<Certificate> {
     Object.is(a.domainName, b.domainName) &&
     Object.is(a.inUse, b.inUse) &&
     Object.is(a.renewalEligibility, b.renewalEligibility) &&
-    Object.is(a.status, b.status);
+    Object.is(a.status, b.status) &&
+    Object.is(a.region, b.region);
```

Comment:
I noticed that the `entityId` method is only using the ARN for the cloud ID, so this addition here is correct, but are you sure you want `update` to run on a record that now spans two regions? The code to deal with that is more complex because you need to get two AWS clients, one for the old region and one for the new, to implement the `replace` functionality correctly.

Reply:
Oh, I see we just shove the state in the database back to whatever AWS has defined in the cloud, so it looks like that should still work fine here. Definitely not the "usual" pattern, though.